packages/dnb-eufemia/src/components/upload/__tests__/useUpload.test.tsx

```diff
@@ -0,0 +1,114 @@
+import { act, render } from '@testing-library/react'
+import useUpload from './../useUpload'
+import React, { useEffect } from 'react'
+import createMockFile from './testHelpers'
+import { useEventEmitter } from '../../../../src/shared/component-helper'
+
+describe('useUpload', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+  it('return an empty list', () => {
+    const validationFunction = jest.fn()
+
+    const MockCompoenent = () => {
```

Comment:
```suggestion
    const MockComponent = () => {
```

Not anything important, just a small spelling mistake which repeats itself a few times in this file 🔤